src/Components/App/App.js

```diff
@@ -76,6 +77,12 @@ const App = () => {
             />
             }}
           />
+          <Route render={ () => {
```

Comment:
I love how you implemented a route to the error 505 page! This is very user friendly :)